Jellyfin.Server/Program.cs

```diff
@@ -689,9 +689,9 @@ private static void SetUnixSocketPermissions(IConfiguration startupConfig, strin
             if (!string.IsNullOrEmpty(socketPerms))
             {
                 [DllImport("libc")]
-                static extern int chmod(string pathname, int mode);
+                static extern int Chmod(string pathname, int mode);
 
-                var exitCode = chmod(socketPath, Convert.ToInt32(socketPerms, 8));
+                var exitCode = Chmod(socketPath, Convert.ToInt32(socketPerms, 8));
 
```

Comment:
is this going to work without the DllImport Entrypoint set to "chmod"?

Reply:
Good call, I've never used DllImport so it's very possible I broke it.
Here's a PR to fix the entrypoint: https://github.com/jellyfin/jellyfin/pull/7495